### PR TITLE
2241picture url

### PR DESCRIPTION
**Pull request by mironovandrei**
[EPMLABSBRN-XX](https://jira.epam.com/jira/browse/EPMLABSBRN-XX)

**Description**:

### CODE REVIEW

File: src/main/kotlin/com/epam/brn/service/UrlConversionService.kt [hunk 1]
```diff
@@ -1,23 +1,46 @@
...
             return ""
-        return awsConfig.baseFileUrl + noiseUrl
+        return cloudService.baseFileUrl() + noiseUrl
     }
 
     fun makeUrlForSubGroupPicture(subGroupCode: String): String =
-        awsConfig.baseFileUrl + folderForThemePictures + "/" + subGroupCode + ".svg"
+        cloudService.baseFileUrl() + folderForThemePictures + "/" + subGroupCode + ".svg"
+
+    fun makeUrlForTaskPicture(pictureFileUrl: String?): String {
```
Comment (ElenaSpb) [inline src/main/kotlin/com/epam/brn/service/UrlConversionService.kt:30]: there we should get as input
pictureFileUrl -> word
to not parse it later

Comment (ElenaSpb) [reply src/main/kotlin/com/epam/brn/service/UrlConversionService.kt:30]: urlConversionService.makeUrlForTaskPicture(resource.pictureFileUrl) 
->
urlConversionService.makeUrlForTaskPicture(resource.word)

Comment (mironovandrei) [reply src/main/kotlin/com/epam/brn/service/UrlConversionService.kt]: Fixed
Now picture url is build based on word

File: src/main/kotlin/com/epam/brn/service/UrlConversionService.kt [hunk 2]
```diff
@@ -1,23 +1,40 @@
...
     fun makeUrlForSubGroupPicture(subGroupCode: String): String =
-        awsConfig.baseFileUrl + folderForThemePictures + "/" + subGroupCode + ".svg"
+        cloudService.baseFileUrl() + folderForThemePictures + "/" + subGroupCode + ".svg"
 
-    fun makeUrlForTaskPicture(pictureTaskUrl: String?): String =
-        awsConfig.baseFileUrl + "/" + pictureTaskUrl?.replace("jpg", "png")?.lowercase()
+    fun makeUrlForTaskPicture(word: String): String {
+        listOf(defaultPicturesPath, unverifiedPicturesPath).forEach { picturesPath ->
+            pictureExtensions.forEach { ext ->
+                val fileName = word.plus(".").plus(ext)
```
Comment (ElenaSpb) [inline src/main/kotlin/com/epam/brn/service/UrlConversionService.kt:33]: let's make it more Kotlinish:

val fileName = "$word.$ext"

?

Comment (mironovandrei) [reply src/main/kotlin/com/epam/brn/service/UrlConversionService.kt]: made assignment more kotlininish

**Pull request merged**